lib/scarpe.rb

```diff
@@ -20,8 +20,8 @@
 
 class Scarpe
   class << self
-    def app(opts = {}, &blk)
-      app = Scarpe::App.new(opts, &blk)
+    def app(...)
```

Comment:
📝 📌 This argument forwarding syntax isn't available until Ruby 2.7 but you have 2.6 in your gemspec. I think it's fine to target 2.7 since 2.6 isn't supported anymore but then we should update the gemspec's `required_ruby_version`

Reply:
Good idea. Yeah we'd mentioned (maybe just before hackdays?) about going hardcore and targeting ruby 3.2. I'll fire something up to do that. thanks